tests/programs/custom/cv32e40x_csr_access_test/cv32e40x_csr_access_test.S

```diff
@@ -401,455 +401,5831 @@ _start0:
 	bne x7, x14, csr_fail
-	csrrwi x14, 769, 0b01001
-	li x7, 0x40801104
+	csrrwi x14, 827, 0b01011
+	li x7, 0x00000000
 	bne x7, x14, csr_fail
-	csrrsi x14, 769, 0b00101
-	li x7, 0x40801104
+	csrrsi x14, 827, 0b00101
+	li x7, 0x00000000
 	bne x7, x14, csr_fail
-	csrrsi x14, 769, 0b11010
-	li x7, 0x40801104
+	csrrsi x14, 827, 0b11010
+	li x7, 0x00000000
 	bne x7, x14, csr_fail
-	csrrsi x14, 769, 0b11011
-	li x7, 0x40801104
+	csrrsi x14, 827, 0b00000
+	li x7, 0x00000000
 	bne x7, x14, csr_fail
-	csrrci x14, 769, 0b00101
-	li x7, 0x40801104
+	csrrci x14, 827, 0b00101
+	li x7, 0x00000000
 	bne x7, x14, csr_fail
-	csrrci x14, 769, 0b11010
-	li x7, 0x40801104
+	csrrci x14, 827, 0b11010
+	li x7, 0x00000000
 	bne x7, x14, csr_fail
-	csrrci x14, 769, 0b10101
-	li x7, 0x40801104
+	csrrci x14, 827, 0b00001
+	li x7, 0x00000000
 	bne x7, x14, csr_fail
-	# mie
+	# mhpmevent28
 	li x7, 0xa5a5a5a5
-	csrrw x14, 772, x7
+	csrrw x14, 828, x7
 	li x7, 0x00000000
 	bne x7, x14, csr_fail
 	li x7, 0x5a5a5a5a
-	csrrw x14, 772, x7
-	li x7, 0xa5a50080
+	csrrw x14, 828, x7
+	li x7, 0x00000000
 	bne x7, x14, csr_fail
-	li x7, 0xf29a87df
-	csrrw x14, 772, x7
-	li x7, 0x5a5a0808
+	li x7, 0x61b52372
+	csrrw x14, 828, x7
+	li x7, 0x00000000
 	bne x7, x14, csr_fail
 	li x7, 0xa5a5a5a5
-	csrrs x14, 772, x7
-	li x7, 0xf29a0088
+	csrrs x14, 828, x7
+	li x7, 0x00000000
 	bne x7, x14, csr_fail
 	li x7, 0x5a5a5a5a
-	csrrs x14, 772, x7
-	li x7, 0xf7bf0088
+	csrrs x14, 828, x7
+	li x7, 0x00000000
 	bne x7, x14, csr_fail
-	li x7, 0x4a27d4c4
-	csrrs x14, 772, x7
-	li x7, 0xffff0888
+	li x7, 0x75526f3b
+	csrrs x14, 828, x7
+	li x7, 0x00000000
 	bne x7, x14, csr_fail
 	li x7, 0xa5a5a5a5
-	csrrc x14, 772, x7
-	li x7, 0xffff0888
+	csrrc x14, 828, x7
+	li x7, 0x00000000
 	bne x7, x14, csr_fail
 	li x7, 0x5a5a5a5a
-	csrrc x14, 772, x7
-	li x7, 0x5a5a0808
+	csrrc x14, 828, x7
+	li x7, 0x00000000
 	bne x7, x14, csr_fail
-	li x7, 0xb1ee3d09
-	csrrc x14, 772, x7
+	li x7, 0xb85c3e25
+	csrrc x14, 828, x7
 	li x7, 0x00000000
 	bne x7, x14, csr_fail
-	csrrwi x14, 772, 0b00101
+	csrrwi x14, 828, 0b00101
 	li x7, 0x00000000
 	bne x7, x14, csr_fail
-	csrrwi x14, 772, 0b11010
+	csrrwi x14, 828, 0b11010
 	li x7, 0x00000000
 	bne x7, x14, csr_fail
-	csrrwi x14, 772, 0b10010
-	li x7, 0x00000008
+	csrrwi x14, 828, 0b00000
+	li x7, 0x00000000
 	bne x7, x14, csr_fail
-	csrrsi x14, 772, 0b00101
+	csrrsi x14, 828, 0b00101
 	li x7, 0x00000000
 	bne x7, x14, csr_fail
-	csrrsi x14, 772, 0b11010
+	csrrsi x14, 828, 0b11010
 	li x7, 0x00000000
 	bne x7, x14, csr_fail
-	csrrsi x14, 772, 0b10010
-	li x7, 0x00000008
+	csrrsi x14, 828, 0b11101
+	li x7, 0x00000000
 	bne x7, x14, csr_fail
-	csrrci x14, 772, 0b00101
-	li x7, 0x00000008
+	csrrci x14, 828, 0b00101
+	li x7, 0x00000000
 	bne x7, x14, csr_fail
-	csrrci x14, 772, 0b11010
-	li x7, 0x00000008
+	csrrci x14, 828, 0b11010
+	li x7, 0x00000000
 	bne x7, x14, csr_fail
-	csrrci x14, 772, 0b00101
+	csrrci x14, 828, 0b00001
 	li x7, 0x00000000
 	bne x7, x14, csr_fail
-	# mtvec
```

Comment:
I have not explicitly disabled this. CLIC is disabled by default. Let me know if we want to change this.

Reply:
We probably would want to run this with clic

Reply:
Added.